src/actions/channels.js

```diff
@@ -181,6 +181,10 @@ export function createGroupChannel(userIds: Array<string>): ActionFunc {
             last_update_at: created.create_at,
         };
 
+        if (created.create_at !== created.update_at) {
+            member = await Client4.getMyChannelMember(created.id);
```

Comment:
I think you'll need to wrap this in a try catch, if the request fails this will throw

Reply:
Yes, good point

Reply:
Done